pkg/cmd/render/render.go

```diff
@@ -279,10 +264,10 @@ func isDisabledViaConfigmap() bool {
 }
 
 type basicInstallConfig struct {
-	CredentialsMode string `yaml:"credentialsMode"`
+	CredentialsMode operatorv1.CloudCredentialsMode `json:"credentialsMode,omitempty"`
```

Comment:
I don't think that you need "omitempty" here. That is only useful for marshaling not for unmarshaling.